test/server/cards/03_TWI/leaders/AsajjVentressUnparalleledAdversary.spec.ts

```diff
@@ -0,0 +1,114 @@
+describe('Asajj Ventress, Unparalleled Adversary', function () {
+    integration(function (contextRef) {
+        describe('Asajj Ventress\'s leader ability', function () {
+            beforeEach(function () {
+                contextRef.setupTest({
+                    phase: 'action',
+                    player1: {
+                        hand: ['smugglers-aid'],
+                        spaceArena: ['green-squadron-awing'],
+                        groundArena: ['battlefield-marine'],
+                        leader: 'asajj-ventress#unparalleled-adversary',
+                        resources: 3,
+                    },
+                    player2: {
+                        groundArena: ['admiral-yularen#advising-caution'],
+                    },
+                });
+            });
+
+            it('should initiate attack but does not give +1/+0 as we do not play any event this phase', function () {
+                const { context } = contextRef;
+
+                // initiate attack
```

Comment:
Can we have ourself play a unit and the opponent play an event to make sure that neither one triggers the ability. The undeployed and deployed test should be pretty much the same.